Debugger/src/Agent.php

```diff
@@ -87,18 +93,22 @@ class Agent
      *            logpoint records. **Defaults to** a new Stackdriver logger.
      *      @type array $daemonOptions Additional options to provide to the
      *            Daemon when registering.
+     *      @type int $maxDepth Limits the number of stackframes with
```

Comment:
Super nit: Do you want to allow INF here?

Reply:
Do you mean specifically the `INF` constant or just the ability to remove the limit?

Reply:
I think INF is not `int`. The type annotation restrict it to `int`. Do you want to relax?

Reply:
Ah, I didn't realize `INF` is a `float`. We can use `PHP_INT_MAX` instead.

Reply:
Should the docs mention how to remove the limit?

Reply:
Yes, probably. I'll add it.

Reply:
Added.